hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/ModifyTableProcedure.java

```diff
@@ -133,8 +133,8 @@ protected void preflightChecks(MasterProcedureEnv env, Boolean enabled) throws H
           "Cannot add or remove column families when this modification " + "won't reopen regions.");
       }
       if (
-        this.unmodifiedTableDescriptor.getCoprocessorDescriptors().hashCode()
-            != this.modifiedTableDescriptor.getCoprocessorDescriptors().hashCode()
+        !new HashSet<>(this.unmodifiedTableDescriptor.getCoprocessorDescriptors())
```

Comment:
This is not on the critial read/write path so maybe this is acceptable, but a better way, is to first check size, and then, create a HashSet for one of the descriptors, and then iterator over the other one, and check whethere the HashSet contains it.